checks/collector.py

```diff
@@ -380,12 +380,15 @@ def run(self, checksd=None, start_event=True, configs_reloaded=False):
             if res:
                 metrics.extend(res)
 
+        # Use `info` log level for some messages on the first run only, then `debug`
+        log_method = log.info if self._is_first_run() else log.debug
+
         # checks.d checks
         check_statuses = []
         for check in self.initialized_checks_d:
             if not self.continue_running:
                 return
-            log.info("Running check %s" % check.name)
+            log_method("Running check %s" % check.name)
```

Comment:
Could you also fix the `%` and use the log interpolation ? `log_method("Running check %s, check.name)`